vulnerabilities/importers/osv.py

```diff
@@ -10,21 +10,26 @@
 import json
 import logging
 from typing import Iterable
+from typing import Iterator
 from typing import List
 from typing import Optional
+from typing import Tuple
 
 import dateparser
 from cvss.exceptions import CVSS3MalformedError
 from cvss.exceptions import CVSS4MalformedError
 from packageurl import PackageURL
+from packageurl.contrib.url2purl import url2purl
 from univers.version_range import RANGE_CLASS_BY_SCHEMES
 from univers.versions import InvalidVersion
 from univers.versions import SemverVersion
 from univers.versions import Version
+from websockets.version import commit
```

Comment:
Why do we use websockets?

Reply:
I’m not sure why the IDE imported this automatically (it’s not used in the code).